src/load/azext_load/aaz/latest/load/_delete.py

```diff
@@ -0,0 +1,165 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "load delete",
+    confirmation="Are you sure you want to perform this operation?",
+)
+class Delete(AAZCommand):
+    """Delete a load test resource.
+
+    :example: Delete Azure load testing resource
+        az load delete --name sample-resource --resource-group sample-rg
+        az load delete --ids /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sample-rg/providers/Microsoft.LoadTestService/loadtests/sample-resource1 /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sample-rg2/providers/Microsoft.LoadTestService/loadtests/sample-resource2
```

Comment:
recommend separating the following two commands into 2 examples

Reply:
Updated it, as two examples